setup.py

```diff
@@ -161,7 +176,7 @@ def run(self):
 
 
 setuptools.setup(
-    py_modules=[py_mod_name],
+    packages=[pkg_name],
```

Comment:
`packages=find_packages()` ([`from setuptools import find_packages`](https://setuptools.readthedocs.io/en/latest/userguide/package_discovery.html#using-find-or-find-packages)) is slightly more future proof (i.e. if a submodule/folder is ever added, it will be autodetected)

Reply:
Thanks, added this. Will push after fixing the build errors.